src/components/filters/Filters.tsx

```diff
@@ -1,3 +1,9 @@
+import { SortRadioGroup } from './SortRadioGroup';
```

Comment:
🟡 import statement could be set to `@/components/filters/SortRadioGroup` to maintain code style consistency

Reply:
I'll fix this